src/main/java/uk/gov/justice/probation/courtcaseservice/controller/CourtCaseController.java

```diff
@@ -105,6 +107,15 @@ Mono<ExtendedCourtCaseRequestResponse> updateCourtCaseByHearingId(@PathVariable(
                 .map(ExtendedCourtCaseRequestResponse::of);
     }
 
+    @Operation(description = "Returns a case's history.")
+    @GetMapping(value = "/cases/{caseId}", produces = MediaType.APPLICATION_JSON_VALUE)
+    @ResponseStatus(HttpStatus.OK)
+    public @ResponseBody
+    CourtCaseHistory getCaseHistory(@PathVariable(value = "caseId") String caseId) {
```

Comment:
Is this the same as getCase?

Reply:
It carries all the updates on the case/hearing as well. @jhevans agree? or should we call it `/cases/{caseId}/history`